lib/urbanopt/geojson/mapper_classes.rb

```diff
@@ -34,33 +34,28 @@
 module URBANopt
     module GeoJSON
         class Mapper < MapperBase
-            # class level variables
             @@instance_lock = Mutex.new
             @@osw = nil
 
             def initialize()
-
```

Comment:
These comments seem valuable for future editors, as long as they're not being read by rdocs.

Reply:
@vtnate any comment within the code gets read by the rdocs